cmd/telegraf/telegraf.go

```diff
@@ -392,6 +412,27 @@ func main() {
 
 	// switch for flags which just do something and exit immediately
 	switch {
+	case *fDeprecationList:
+		c := config.NewConfig()
+		infos := c.CollectDeprecationInfos(
+			inputFilters,
+			outputFilters,
+			aggregatorFilters,
+			processorFilters,
+		)
+		//nolint:revive // We will notice if Println fails
+		fmt.Println("Deprecated Input Plugins: ")
```

Comment:
@reimda @srebhan I know its a little late, but just came to this `nolint` tag. Is there a reason we can't use this?

```suggestion
		_, _ = fmt.Println("Deprecated Input Plugins: ")
```